lips/lip-6.mdx

```diff
@@ -0,0 +1,140 @@
+---
+lip: 6
+title: Libra Validator Configuration Management
+authors: Valeria Nikolaenko, David Wolinsky, David Dill
+status: Draft
+type: Informational
+created: 07/30/2020
+---
+# Summary
+This LIP describes the conceptual governance model of the Libra blockchain, it discusses the management of its main component - the on-chain validator set configuration.
+Validator set is part of a global on-chain configuration that additionally includes the list of registered currencies, publishing policies and Libra version.
+
+# Abstract / Overview
+The Libra Payment Network (“LPN”) is a blockchain-backed payment platform and an infrastructure for stablecoins. Both the blockchain and the stablecoin financial instruments are controlled by Libra Networks (“Libra Networks” or “LN”) a subsidiary of the Libra Association (the “Association” or “LA”). Libra Networks is in the process of applying for a payment system license from the Swiss Financial Market Supervisory Authority (“FINMA”). Once granted, LN will be responsible for ensuring that LPN operates in compliance with the payment system license from FINMA (the "FINMA License").
```

Comment:
Let's say LibraRoot which is a shared entity on chain between LA and LPN and use LibraRoot instead of association hereafter